.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will launch at 00:00 every Sunday
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
```

Comment:
Is this line needed?

Reply:
here it is useful. since we can manually trigger it and don't have to wait for next Sunday.